### PR TITLE
Refactor context manager

### PR DESCRIPTION
**Pull request by shshshl09**
* Remove unused code
* Add FORCE_SET_TOKEN const
* Replace namespaceNameToStateInfo with namespaceToNameStateInfo

### CODE REVIEW

File: nugu-core/src/main/java/com/skt/nugu/sdk/core/context/ContextManager.kt [hunk 1]
```diff
@@ -86,7 +76,7 @@ class ContextManager : ContextManagerInterface {
         stateRequestToken: Int
     ): ContextSetterInterface.SetStateResult {
         lock.withLock {
-            if (stateRequestToken == 0) {
+            if (stateRequestToken == ContextSetterInterface.FORCE_SET_TOKEN) {
```
Comment (Yuseon-Han) [inline nugu-core/src/main/java/com/skt/nugu/sdk/core/context/ContextManager.kt:79]: It never happen

File: nugu-core/src/main/java/com/skt/nugu/sdk/core/context/ContextManager.kt [hunk 2]
```diff
@@ -183,7 +177,7 @@ class ContextManager : ContextManagerInterface {
         timeoutInMillis: Long
     ) {
         lock.withLock {
-            if (stateRequestToken == 0) {
+            if (stateRequestToken == ContextSetterInterface.FORCE_SET_TOKEN) {
```
Comment (Yuseon-Han) [inline nugu-core/src/main/java/com/skt/nugu/sdk/core/context/ContextManager.kt:180]: It never happen
take look stateRequestToken initial value

File: nugu-core/src/main/java/com/skt/nugu/sdk/core/context/ContextManager.kt (hunk 1 above)
Comment (shshshl09) [reply nugu-core/src/main/java/com/skt/nugu/sdk/core/context/ContextManager.kt:79]: see followings:
* refreshAllState()
* DefaultAudioPlayerAgent.changeActivity's executeProvideState(..., 0)
* ...

"0" value was used to update the state without a request.

Comment (shshshl09) [reply nugu-core/src/main/java/com/skt/nugu/sdk/core/context/ContextManager.kt:79]: Commit related: f48b722

Comment (Yuseon-Han) [reply nugu-core/src/main/java/com/skt/nugu/sdk/core/context/ContextManager.kt:79]: I have confused with variable stateRequestToken.
Okay I see.

File: nugu-core/src/main/java/com/skt/nugu/sdk/core/context/ContextManager.kt (hunk 2 above)
Comment (Yuseon-Han) [reply nugu-core/src/main/java/com/skt/nugu/sdk/core/context/ContextManager.kt:180]: I've just understand that this is not initial value check, but using Integer Overflow.
:)

**Review by Yuseon-Han — APPROVED** [review]

File: nugu-core/src/main/java/com/skt/nugu/sdk/core/context/ContextManager.kt (hunk 2 above)
Comment (shshshl09) [reply nugu-core/src/main/java/com/skt/nugu/sdk/core/context/ContextManager.kt:181]: Commit : 5036996

**Pull request merged**